bisqapps/shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/data/model/UserProfile.kt

```diff
@@ -0,0 +1,14 @@
+package network.bisq.mobile.domain.data.model
+
+// TODO: Is it okay for the models to be mutable?
```

Comment:
I guess the question refers to the `open` ..? Unless there is a really strong inheritance relationship, its better to use composition.